src/Platform/App/components/Map/MapContainer.tsx

```diff
@@ -252,42 +364,41 @@ export const MapContainer = () => {
     const allYearInputs: RequestParamsAllYears = {
       facilityLat: facilityCoordinates.lat,
       facilityLng: facilityCoordinates.lng,
-      transmission: {
-        VoltageClass: '230 kV Single Circuit',
-        ConductorType: 'ACSS',
-        Structure: 'Lattice',
-        LengthCategory: '> 10 miles',
-        NewOrReconductor: 'New',
-        AverageTerrainMultiplier: 1,
-        Miles: {
-          Forested: 0,
-          Flat: 0,
-          Wetland: 0,
-          Farmland: 0,
-          Desert: 0,
-          Urban: 0,
-          Hills: 0,
-          Mountain: 0,
-          Zone1: 1,
-          Zone2: 0,
-          Zone3: 0,
-          Zone4: 0,
-          Zone5: 0,
-          Zone6: 0,
-          Zone7: 0,
-          Zone8: 0,
-          Zone9: 0,
-          Zone10: 0,
-          Zone11: 0,
-          Zone12: 0
-        }
-      },
+      transmission: DEFAULT_TRANSMISSION_VAL,
       teaModel: teaModel,
       teaInputs: teaInputs
     };
 
     let continueProcessing = true;
 
+    const allYearInputsStr = createQueryStr(allYearInputs);
+    const biomassCoordinatesStr = createQueryStr(biomassCoordinates);
+    const frcsInputsStr = createQueryStr(frcsInputs);
+    const transportInputsStr = createQueryStr(transportInputs);
+
+    const shortenUrl = await fetch(
+      `${baseUrl}/shorten-url`,
+      {
+        mode: 'cors',
+        method: 'POST',
+        body: JSON.stringify({allYearInputsStr, biomassCoordinatesStr, frcsInputsStr, transportInputsStr}),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }
+    )
```

Comment:
similar to my comment in the Db proj PR -- since you aren't actually going with query strings anymore, why not just pass this as regular JSON instead of separate stringified JSON objects?  You could just make them props inside a container object and store them in matching JSON columns in PGSQL.  Not really a huge perf win but less to worry about and if we ever wanted to search the db for something, like most common frcs data or something, would make it possible.

Reply:
I have made the following changes. Instead of separate data fields I'm passing a single object which will be stored as a JSONB `body: JSON.stringify({ data }). .` However, I still need to stringify the data as I'm getting a TS error which needs the req body to be a BodyInt type.